src/Sales.cpp

```diff
@@ -0,0 +1,121 @@
+#include <fstream>
```

Comment:
Sorry I didn't catch this last review, but usually source files just include the associated header, and the header includes everything else.

So, move these includes to ```Sales.hpp``` and just keep the ```#include "Sales.hpp"``` include.